src/components/UserBubble/UserBubble.vue

```diff
@@ -0,0 +1,146 @@
+<!--
+  - @copyright Copyright (c) 2019 Jonas Sulzer <jonas@violoncello.ch>
+  -
+  - @author Marco Ambrosini <marcoambrosini@pm.me>
+  - @author Jonas Sulzer <jonas@violoncello.ch>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<docs>
+
+### General description
+
+This component displays a user together with a small avatar in a grey bubble. 
+It's possible to replace the avatar with something else, 
+to link the bubble to e.g. a users profile 
+and to show a popover on hover with e.g. the full user name handle / email address or something else.
+
+This components has two slots:
+* 'avatar' which allows you to replace the user-avatar with e.g. an icon;
+
+* a default slot which is for the content of the popover (this is passed to the popover component directly).
+
+### Examples
+
+With a `<button>` as a trigger:
+```vue
+<template>
+	<p>
+		Some text before
+		<user-bubble :user="'admin'" :displayName="'Admin Example'" :url="'/test'" >
+  			@admin@foreign-host.com
+		</user-bubble>
+		 and after the bubble.
+	</p>
+</template>
+```
+</docs>
+
+<template>
+	<Popover trigger="hover focus" :open="open" :disabled="popoverEmpty"
+		class="user-bubble-popover" @update:open="onOpenChange">
+		<div slot="trigger" v-bind="linkOrNot" class="user-bubble">
+			<slot name="avatar">
```

Comment:
Let's use a prop and detect if it's a url instead of an avatar
So it can be a css class or just a plain url

Reply:
implemented!